### DISCUSSION TITLE
Large compiled main.js

### DISCUSSION THREAD

**Discussion by jvsteiner** [markdown-confluence/markdown-confluence#64] [General]
more of a question than an issue, but I noticed that the "compiled" size of the main.js file is currently around 20 MB - that's pretty heavy.  Do you know why it is so large?  Probably not worth worrying about for now, but maybe at some point it could be optimized.

**Comment by andymac4182** [comment]
It has to include everything in one file. A lot of the file is mermaid.js to render mermaid to an image. This is base64ed so it's quite large. It also includes ADF renderer for local previews of the  conversion. 

I will have a look at how I can make the file smaller.

**Comment by andymac4182** [comment]
Reduced to 9MB currently. Lots better than the 20MB :)
